mypy complains about leoserver.py with Python 3.11

See PR #3698. (PR #3694 fixed complaints only for Python 3.12).

The API for `ConnectionClosed` is [here](https://websockets.readthedocs.io/en/stable/reference/exceptions.html#websockets.exceptions.ConnectionClosed).

mypy issues the following complaint for Python 3.11.
```console
leo\core\leoserver.py:5449: error: Missing positional argument "sent" in call to "ConnectionClosed"  [call-arg]
Found 1 error in 1 file (checked 260 source files)
```
Nevertheless, leoInteg works with all versions of Python.

--- a/leo/core/leoserver.py
+++ b/leo/core/leoserver.py
@@ -5445,10 +5445,9 @@ def main() -> None:  # pragma: no cover (tested in client)
                     elif trace:
                         print(f"{tag}: got: {d}", flush=True)
                     answer = controller._do_message(d)
-                except TerminateServer as e:
-                    raise websockets.exceptions.ConnectionClosed(
-                        websockets.frames.Close(websockets.frames.CloseCode.NORMAL_CLOSURE,
-                                                e.__str__()))
+                except TerminateServer as e:  ###
+                    rcvd = websockets.frames.Close(websockets.frames.CloseCode.NORMAL_CLOSURE, e.__str__())
+                    raise websockets.exceptions.ConnectionClosed(rcvd, None, None)
                 except ServerError as e:
                     data = f"{d}" if d else f"json syntax error: {json_message!r}"
                     error = f"{tag}:  ServerError: {e}...\n{tag}:  {data}"

--- a/leo/core/leoserver.py
+++ b/leo/core/leoserver.py
@@ -5445,7 +5445,7 @@ def main() -> None:  # pragma: no cover (tested in client)
                     elif trace:
                         print(f"{tag}: got: {d}", flush=True)
                     answer = controller._do_message(d)
-                except TerminateServer as e:  ###
+                except TerminateServer as e:
                     rcvd = websockets.frames.Close(websockets.frames.CloseCode.NORMAL_CLOSURE, e.__str__())
                     raise websockets.exceptions.ConnectionClosed(rcvd, None, None)
                 except ServerError as e:

